manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Google.PlatformTools
+PackageVersion: 35.0.2
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: platform-tools\adb.exe
+  PortableCommandAlias: adb
+- RelativeFilePath: platform-tools\fastboot.exe
+  PortableCommandAlias: fastboot
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.google.com/android/repository/platform-tools_r35.0.2-win.zip
+  InstallerSha256: 2975A3EAC0B19182748D64195375AD056986561D994FFFBDC64332A516300BB9
+  ArchiveBinariesDependOnPath: true
+ManifestType: installer
+ManifestVersion: 1.9.0

```

Comment:
```suggestion
ManifestVersion: 1.7.0
```

Reply:
Manifest 1.9.0 is [live](https://github.com/microsoft/winget-pkgs/pull/195988).